pages/about.html

```diff
@@ -0,0 +1,21 @@
+    <div id="about" class="container-fluid">
+        <div class="row">
+          <div class="col-xs-12 col-sm-9 col-sm-offset-1 col-md-9 col-md-offset-1">
+            <div class="text-center">
+                <h1 class="head-one">ABOUT ME</h1>
+                <h3 class="head-two">I'M A CREATIVE WEB DEVELOPER</h3>
+                <hr style="color:#555566">
+            </div>
+            
+            <div class="col-xs-12 col-sm-12 col-md-8" style="float: left">
+                <p style="text-align: justify"> 
+                 An ambitious problem solver with a passion for online businesses, has extensive analysis knowledge and much experience of creating logical and innovative solutions to complex problems to increase efficiency.</p>
+                <p style="text-align: justify"> As a web developer I have a keen interest in technology, mobile applications and user experience. As someone who takes responsibility for his own personal development, I am continually evaluating and upgrading the skills so that I stay at the cutting edge of web development.</p>
+            </div>
+            
+            <div class="col-md-4 col-sm-12 col-xs-12">                
+                <img id="my-img" class="center-block" src="/asset/img/person.jpg" alt="my_photo">
```

Comment:
When src attributes starts with "/" it is considered as an absolute path starting from domain root. This will not work if you host several application in one domain:
Say instead of http://sergey.com/index.html you would move your app to http://sergey.com/myresume/index.html.
I'd recommend to use relative path in this case: "asset/img/person.jpg"